Source/Core/Core/PowerPC/JitArm64/JitArm64_FloatingPoint.cpp

```diff
@@ -339,8 +339,12 @@ void JitArm64::fctiwzx(UGeckoInstruction inst)
   }
   else
   {
-    m_float_emit.FCVT(32, 64, EncodeRegToDouble(VD), EncodeRegToDouble(VB));
-    m_float_emit.FCVTS(EncodeRegToSingle(VD), EncodeRegToSingle(VD), ROUND_Z);
+    ARM64Reg V1 = gpr.GetReg();
+
+    m_float_emit.FCVTS(V1, EncodeRegToDouble(VB), ROUND_Z);
+    m_float_emit.FMOV(EncodeRegToSingle(VD), V1);
```

Comment:
Why do you need a temp register here?
```
    m_float_emit.FCVTS(EncodeRegToSingle(VD), EncodeRegToDouble(VB), ROUND_Z);
    m_float_emit.FMOV(EncodeRegToSingle(VD), EncodeRegToSingle(VD));
```

Reply:
My understanding is that the FCVT*S instruction family supports:
- f64-to-s32 with the result in a general purpose register
- f64-to-s64 with the result in a SIMD&FP register
but we need f64-to-s32 and the result in a SIMD&FP register.

Reply:
Oh, I have not noticed that this temporary register was a GPR one, my bad.